packages/react-storage/src/components/StorageBrowser/views/LocationActionView/DeleteView/DeleteView.tsx

```diff
@@ -53,19 +53,34 @@ export const DeleteView = ({
         onActionStart={onActionStart}
         onActionCancel={onActionCancel}
       >
-        <Exit onClick={onExit} disabled={isProcessing} />
+        <ViewElement className={`${AMPLIFY_CLASS_BASE}__navigation`}>
```

Comment:
Should this classname denote that it's a container?

Reply:
I don't think so

Reply:
Got it, when should a wrapping component have "wrapper" (or something similar in the classname)?